code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -203,6 +203,12 @@
 	icon_state = "satchel"
 	burn_state = FIRE_PROOF
 
+/obj/item/weapon/storage/backpack/satchel/wizard
+	name = "scroll case"
+	desc = "Made from the leather of \"normies\", it is superior for carrying your wizardly needs."

```

Comment:
It's not made from just assistants. It's made from any sort of non-magical user. IE: Normal People. Which shortens into the meme, "normies".